specification/compute/resource-manager/Microsoft.Compute/stable/2018-04-01/compute.json

```diff
@@ -7475,6 +7475,10 @@
         "singlePlacementGroup": {
           "type": "boolean",
           "description": "When true this limits the scale set to a single placement group, of max size 100 virtual machines."
+        },
+        "proximityPlacementGroup": {
+          "$ref": "#/definitions/SubResource",
+          "description": "Specifies information about the proximity placement group that the virtual machine scale set should be assigned to. <br><br>Minimum api-version: 2018-04-01."
```

Comment:
is `<br><br>` intentional?

Reply:
Yes, this is in sync with the other properties where we show minimum api-version as part of property description

Reply:
ideally this should be added in a new api-version. But since for Compute, PUT is a PATCH . it will not (should not) be a problem